github/actions_secrets.go

```diff
@@ -66,42 +62,44 @@ func (s *ActionsService) GetRepoPublicKey(ctx context.Context, owner, repo strin
 	return pubKey, resp, nil
 }
 
+// GetRepoPublicKey gets a public key that should be used for secret encryption.
+//
+// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions/#get-a-repository-public-key
+func (s *ActionsService) GetRepoPublicKey(ctx context.Context, owner, repo string) (*PublicKey, *Response, error) {
+	url := fmt.Sprintf("repos/%v/%v/actions/secrets/public-key", owner, repo)
+	return s.getPublicKey(ctx, url)
+}
+
+// GetRepoDependabotPublicKey gets a public key that should be used for secret encryption.
+//
+// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions/#get-a-repository-public-key
```

Comment:
Is there a better URL for this endpoint?

Reply:
I was going to say that this is an undocumented API, but docs were posted last night! https://github.com/dependabot/dependabot-core/issues/3297#issuecomment-1009308206